docs/12-design.md

```diff
@@ -55,23 +55,22 @@ sometimes be used to avoid reading a lot of the data).
 ## Tables
 
 All records in a table conform to a schema. The records in a table are stored in multiple files, with each file
-belonging to a partition. These files are all in an S3 bucket that is exclusively used by this table.
+belonging to a partition. These files are all stored in an S3 bucket deployed at an instance level, referred to as 
+the data bucket.
 
 Each table has a state store associated to it. This stores metadata about the table, namely the files that are in
 the table and how the records in the table are partitioned.
 
-Tables are deployed by the CDK table stack. This stack creates the infrastructure for each table. Each table
-requires a bucket where its data will be stored, and a state store. When a table is first created, its state store
-must be initialised.
+Table infrastructure is deployed at an instance level by the CDK table stacks. These stacks creates the infrastructure 
+for all tables. All tables require an S3 bucket to store data and a state store to store metadata about files in each table. 
+When a table is first created, its state store must be initialised. To achieve this, both state store implementations 
+support storing metadata for files in all tables. 
```

Comment:
Can we make it clear that this is all deployed for the instance rather than the tables? There's some ambiguity in "all tables require an S3 bucket and a state store". It might be best to start by explaining what is deployed for an instance, and then how tables use that infrastructure.